src/main/kotlin/de/smartsquare/socketio/emitter/LettucePublisher.kt

```diff
@@ -1,9 +1,9 @@
 package de.smartsquare.socketio.emitter
 
-import io.lettuce.core.api.StatefulRedisConnection
+import io.lettuce.core.api.sync.RedisCommands
 
-class LettucePublisher(private val connection: StatefulRedisConnection<Any, Any>) : RedisPublisher {
+class LettucePublisher(private val commands: RedisCommands<String, String>) : RedisPublisher {
```

Comment:
For lettuce, the consumer can pass in a `Codec` that encodes and decodes what is sent to redis. So you could also have e.g. `RedisCommands<ByteArray, ByteArray>`. When using the instance, we need to know what the types are though so we now what we can send and we request an instance that handles Strings. I think it will be that anyways in almost all cases since all default APIs return that.